model_utilities/train/finetuner.py

```diff
@@ -42,53 +44,68 @@ def __plot_losses(self, training_losses: list[float], validation_losses: list[fl
         plt.ylabel("Loss")
         plt.legend()
         plt.savefig("./debug/graphs/losses.png")
-
-    def __train(self, model: AutoModelForSeq2SeqLM, tokenizer: NllbTokenizer, dataset: ds.Dataset, optimizer: Adafactor,
+        
+    def __tokenize(self, dataset: ds.Dataset, src_lang: str, tgt_lang: str, tokenizer: NllbTokenizerFast, max_length : int) -> BatchEncoding:
+        tokenizer.src_lang = src_lang
+        tokenizer.tgt_lang = tgt_lang
+        tokenized_input = tokenizer(dataset[src_lang], text_target=dataset[tgt_lang], return_tensors='pt', padding=True, truncation=True, max_length=max_length)
+        tokenized_input.labels[tokenized_input.labels == tokenizer.pad_token_id] = -100
```

Comment:
Why -100 exactly? Please either move it to a variable and then use it, or explain its meaning in a comment.